tests/xharness/Jenkins.cs

```diff
@@ -2872,8 +2874,9 @@ protected override async Task RunTestAsync ()
 				using (var proc = new Process ()) {
 
 					proc.StartInfo.WorkingDirectory = WorkingDirectory;
-					proc.StartInfo.FileName = "/Library/Frameworks/Mono.framework/Commands/mono";
+					proc.StartInfo.FileName = "xibuild";
 					var args = new StringBuilder ();
+					args.Append ("-t -- ");
```

Comment:
Same: won't this try to write to `/Library/Frameworks/Mono.framework/Commands/mono.config`?

Reply:
This is running `TestExecutable` with `xibuild`, which should be nunit from the packages directory.